### PR TITLE
Adding local notification to remind user app is running

### PR DESCRIPTION
**Pull request by yousefhamza**
In the commit:
- Scheduling the notification
- Canceling the notification
- Dealing with persmissions

Info:
- The notification is Scheduled 4 hours after starting

Signed-off-by: Yousef Hamza jo.adam.93@gmail.com

### CODE REVIEW

File: Subsurface/Subsurface/SAppDelegate.m [hunk 1]
```diff
@@ -21,6 +21,7 @@
 #import "SAppDelegate.h"
 
 #define kAnimationOpacityKey    @"animateOpacity"
+#define HOUR 3600
```
Comment (zdanovs) [inline Subsurface/Subsurface/SAppDelegate.m]: Let's stick to a way of naming defines as in other places among this codebase — starts with lowercase "k" and then a CamelCase

File: Subsurface/Subsurface/SAppDelegate.m [hunk 2]
```diff
@@ -151,4 +163,24 @@ - (void)registerDefaultsFromSettingsBundle {
     [userDefaults synchronize];
 }
 
+- (void)scheduleNotification {
+	self.batteryNotification = [[UILocalNotification alloc] init];
+	self.batteryNotification.fireDate = [NSDate dateWithTimeIntervalSinceNow: 10];
+	self.batteryNotification.alertBody = @"Debugging- 5";
```
Comment (zdanovs) [inline Subsurface/Subsurface/SAppDelegate.m]: @"Debugging- 5" strings seems too technical. Maybe some better string that describes what for it is?

**Comment by zdanovs** [comment]
Something wrong with new line intend. All your new lines are further then original ones.

**Comment by yousefhamza** [comment]
Okay I noticed the "Debugging 5" and I'm just merging it now and and will do the naming also of  'HOURS'.

The indentation is actually correct it's just the editor here that makes it look wider that it should be.

**Pull request merged**